examples/integration_test.py

```diff
@@ -842,6 +857,56 @@ def my_on_revoke(consumer, partitions):
     c.close()
 
 
+def verify_throttle_cb():
+    """ Verify throttle_cb is invoked """
+    conf = {'bootstrap.servers': bootstrap_servers,
+            'api.version.request': api_version_request,
+            'linger.ms': 500,
+            'client.id': 'throttled_client',
+            'throttle_cb': throttle_cb}
+
+    p = confluent_kafka.Producer(conf)
+
+    msgcnt = 1000000
```

Comment:
Do you really need this many messages to trigger the throttling?

Reply:
I guess that depends on how aggressive you set your client quota :)

Reply:
Right, but it is better if the tests are as modest as possible (performance tests excluded of course) to be runnable in CI, whatnot.